home/views.py

```diff
@@ -7,3 +8,29 @@ def about(request):
 
 def landingpage(request):
     return render(request, 'home/landingpage.html')
+
+
+def displayQuestion(request, **kwargs):
+    question = Question.objects.all().get(id=kwargs['pk'])
+    sortAnsBy = request.GET["sortanswersby"] if 'sortanswersby' in request.GET else ''
+    context = create_display_question_context(question, sortAnsBy)
+    return render(request, 'home/question_detail.html', context)
+
+
+def create_display_question_context(question, sortAnsBy):
```

Comment:
why is this function still here? ignoring my comments about it will not make me forget them.

Reply:
I didn't ignore it, I did change and moved some functions to the models.py. Seeing that it's still not good enough, I moved all the context-making login to models.